astronomy/sol_gravity_model.proto.txt

```diff
@@ -194,11 +262,84 @@ gravity_model {
+        column {order:  6, cos: -5.19216872092000e-09, sin: -2.79591282159000e-07},
+        column {order:  7, cos: -1.93675835264000e-07, sin: -4.84129031019000e-07}},
+      row {degree: 8,
+        column {order:  0, cos: -4.25832531474000e-07, sin:  0},
+        column {order:  1, cos: -1.97858005196000e-08, sin: -2.61463193793000e-07},
+        column {order:  2, cos:  1.63477675115000e-07, sin: -2.32108432597000e-07},
+        column {order:  3, cos:  3.06132013243000e-07, sin: -3.93795936874000e-09},
+        column {order:  4, cos: -1.09546245610000e-07, sin:  8.20832304879000e-08},
+        column {order:  5, cos: -3.55484878013000e-08, sin: -1.77529137038000e-08},
+        column {order:  6, cos: -5.92247928748000e-08, sin: -2.97194470268000e-07},
+        column {order:  7, cos: -3.16832604822000e-08, sin: -8.52087760261000e-08},
+        column {order:  8, cos:  9.47582305906000e-08, sin:  2.95091759971000e-07}},
+      row {degree: 9,
+        column {order:  0, cos: -2.35102795624000e-07, sin:  0},
+        column {order:  1, cos: -9.76227225913000e-08, sin:  2.31225333712000e-08},
+        column {order:  2, cos: -2.41388180798000e-07, sin: -1.73278758693000e-07},
+        column {order:  3, cos:  2.58158855146000e-07, sin:  1.40158486106000e-08},
+        column {order:  4, cos: -1.57982225850000e-07, sin:  5.99197635642000e-08},
+        column {order:  5, cos:  5.60326197830000e-08, sin:  2.67783115324000e-08},
+        column {order:  6, cos: -3.47948445859000e-08, sin: -1.91293276916000e-07},
+        column {order:  7, cos: -1.92496903122000e-07, sin:  1.44989557767000e-07},
+        column {order:  8, cos:  9.39439943048000e-08, sin:  1.35982512917000e-07},
+        column {order:  9, cos:  2.52845540672000e-07, sin: -1.63264704004000e-09}},
+      row {degree: 10,
+        column {order:  0, cos: -2.43651086082000e-07, sin:  0},
+        column {order:  1, cos: -1.01519431655000e-07, sin:  7.75741049843000e-08},
+        column {order:  2, cos: -2.83483163202000e-08, sin: -1.35987149871000e-07},
+        column {order:  3, cos: -5.71030640092000e-08, sin:  6.76237977826000e-08},
+        column {order:  4, cos:  3.41263062776000e-07, sin: -5.81873613050000e-08},
+        column {order:  5, cos: -2.92953810089000e-07, sin:  2.27894439060000e-08},
+        column {order:  6, cos: -2.04889687126000e-07, sin:  1.72223321904000e-07},
+        column {order:  7, cos: -1.61605271928000e-07, sin: -7.24300220001000e-08},
+        column {order:  8, cos:  1.39213026564000e-07, sin:  1.22221645955000e-07},
+        column {order:  9, cos:  1.73355586394000e-07, sin: -8.20685718579000e-08},
+        column {order: 10, cos: -4.27187834361000e-08, sin:  1.61075401825000e-08}},
+    }
   }
   body {
-    # J2 coefficient and reference radius from MRO data,
+    # Spherical harmonic coefficients and reference radius from MRO data,
     # http://pds-geosciences.wustl.edu/mro/mro-m-rss-5-sdp-v1/mrors_1xxx/data/shadr/jgmro_110c_sha.tab.
```

Comment:
There is an MRO120D now, see Konopliv et al. (2016), *An improved JPL Mars gravity field and orientation from Mars orbiter and lander tracking data*; we should probably use that.
Data at http://pds-geosciences.wustl.edu/mro/mro-m-rss-5-sdp-v1/mrors_1xxx/data/shadr/jgmro_120d_sha.tab, predictably.

Alternatively, there is GMM-3: Genova et al. (2016), *Seasonal and static gravity field of Mars from MGS, Mars Odyssey and MRO radio science*,
see https://pgda.gsfc.nasa.gov/products/57 (the data is at http://pds-geosciences.wustl.edu/mro/mro-m-rss-5-sdp-v1/mrors_1xxx/data/shadr/gmm3_120_sha.tab).